### ISSUE TITLE
Error messages (40)

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#377]
The following JSON snippet contains a syntax error: A closing curly bracket is missing.

```
"file_name" : "test",
"[import]" : "./dynamic_imports/${file_name.jsonp"
```

Result:

`Invalid expression found: '... /dynamic_imports/${file_name.jsonp' - The double quotes are missing!`

That's wrong. Expected is a syntax error about a missing curly bracket.

Cross check:

`"param" : "${dictParam"`

Result: Error detected properly

`Invalid syntax! One or more than one closed curly bracket is missing in expression '${dictParam'.`

But if I add content inside the quotes:

`"param" : "./${dictParam"`

again the error message is wrong:

`Invalid expression found: './${dictParam' - The double quotes are missing!!!`

**Comment by HolQue** [comment]
JPP_1156

**Comment by HolQue** [comment]
Error message changed to:

`Could not load the import file './dynamic_imports/${AA/imported.AA.jsonp'. The parameter '' is not available!`

But every '`${`' requires a corresponding '`}`'. This is not detected.

**Comment by namsonx** [comment]
Hello Holger,

This ticket was solved on stabi branch.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
merged for version 0.13.1
